### DISCUSSION TITLE
Listar mídias recentes.

### DISCUSSION THREAD

**Discussion by FredericoWP** [Flutterando/forum#78] [Perguntas e Respostas] [answered]
Olá galera !
Durante uma atividade para praticar o carregamento de imagens fiquei com a seguinte dúvida.

Como posso fazer uma lista personalizada com minhas mídias (fotos e vídeos) mais recentes na galeria local?

O esquema da tela consegui fazer, porém não faço ideia como fazer a lista com as mídias. 
Conforme anexo.

![Screenshot_2022-10-28-14-48-46-627_com.example.cygnusgestor.jpg](https://user-images.githubusercontent.com/113041949/201762942-9149ffde-0c5f-4133-9723-140df83752d8.jpg)

**Comment by eduardoflorence** [comment, marked as answer]
Olá @FredericoWP, veja se esse package consegue te ajudar: https://pub.dev/packages/image_list
Ele usa o nativo para fazer a ponte com o Flutter.
